gbus/saga/glue.go

```diff
@@ -256,6 +257,7 @@ func (imsm *Glue) completeOrUpdateSaga(tx *sql.Tx, instance *Instance) error {
 
 	if instance.isComplete() {
 		imsm.Log().WithField("saga_id", instance.ID).Info("saga has completed and will be deleted")
+		metrics.SagaLatencySummary.WithLabelValues(instance.StartedBy, reflect.TypeOf(instance.UnderlyingInstance).String()).Observe(float64(time.Since(instance.CreatedAt) / time.Millisecond))
```

Comment:
@kerenbe4 instance.StartedBy is not the service that is running the saga but rather the service issuing the command or event that triggered the saga.
What you are looking for is imsm.svcName

```suggestion
		metrics.SagaLatencySummary.WithLabelValues(imsm.svcName, reflect.TypeOf(instance.UnderlyingInstance).String()).Observe(float64(time.Since(instance.CreatedAt) / time.Millisecond))
```

Reply:
This is the field that I wanted :) 
I have the saga type which is the one I'm using to filter to see the behavior of a specific saga. 
rethinking it, the labels are confusing, and it can be useful to have also the service name
Thanks